build-logic/src/main/kotlin/api.kt

```diff
@@ -1,15 +1,20 @@
-
 import org.gradle.api.Project
 import org.gradle.api.Task
 import org.gradle.api.tasks.TaskProvider
 import org.gradle.jvm.tasks.Jar
 import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
+import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
 import org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension
 import org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin
 import org.jetbrains.kotlin.gradle.targets.js.npm.tasks.KotlinNpmInstallTask
 
 class AndroidOptions(
-    val withCompose: Boolean
+    val withCompose: Boolean,
+)
+
+class KotlinCompilerOptions(
+    val apiVersion: KotlinVersion = KotlinVersion.KOTLIN_2_0,
+    val languageVersion: KotlinVersion = KotlinVersion.KOTLIN_2_0,
```

Comment:
Nit: is there any occurrence where we're not setting the same value for both parameters? I would probably have gone with a single parameter but 2 work too 👍

Reply:
No you're totally right :) Changed it to a single `version` parameter